model2vec/utils.py

```diff
@@ -49,6 +60,53 @@ def save_pretrained(folder_path: Path, embeddings: np.ndarray, tokenizer: Tokeni
 
     logger.info(f"Saved model to {folder_path}")
 
+    # Optionally create the model card
+    if create_model_card:
+        _create_model_card(folder_path, **kwargs)
+
+
+def _create_model_card(folder_path: Path, **kwargs: Any) -> None:
+    """
+    Create a model card and store it in the specified path.
+
+    :param folder_path: The path where the model card will be stored.
+    :param **kwargs: Additional metadata for the model card (e.g., model_name, base_model, etc.).
+    """
+    folder_path = Path(folder_path)
+    model_name = folder_path.name
+
+    # Generate the model card content
+    model_card = _generate_model_card(model_name, **kwargs)
+
+    # Save the model card as README.md
+    with open(folder_path / "README.md", "w", encoding="utf8") as fOut:
+        fOut.write(model_card)
+
+
+def _generate_model_card(model_name: str, template_path: str = "assets/model_card_template.md", **kwargs: Any) -> str:
+    """
+    Generate a model card using a string template.
+
+    :param model_name: The name of the model.
+    :param template_path: The path to the template file.
+    :param **kwargs: Additional metadata for the model card (e.g. language, base_model, etc.).
+    :return: The content of the model card in markdown format.
+    """
+    # Load the template
+    template_path = pkg_resources.resource_filename("model2vec", "model_card_template.md")
+    with open(template_path, "r") as file:
+        template_content = file.read()
+
+    # # Fill in the placeholders in the template
+    model_card_content = template_content.format(
+        model_name=model_name,
+        base_model=kwargs.get("base_model_name", "unknown"),
```

Comment:
if you expect these arguments, I wouldn't put them in kwargs, but just give them names and these defaults. The behavior is the same, it's just typed better.